webui/src/pages/repositories/wizard/wizard.jsx

```diff
@@ -0,0 +1,106 @@
+import React, {useState} from "react";
+import StepWizard from "react-step-wizard";
+import {ProgressBar} from "react-bootstrap";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+
+const transitionDefaults = {
+    enterRight: "",
+    enterLeft: "",
+    exitRight: "",
+    exitLeft: "",
+    intro: "",
+}
+
+const Wizard = ({
+                    hashEnabled=false,
+                    transitions=transitionDefaults,
+                    progressBar = false,
+                    isGoBackButton = false,
```

Comment:
```suggestion
                    goBackButton = false,
```
or
```suggestion
                    canGoBack = false,
```
?

AFAIU from reading the code, this says whether to _add_ a "go back button", _not_ that this _is" a "go back button".  So it's like `progressBar` IIUC.

Similarly for `isSkipButton`; I'd expect `canSkip` or `skipButton`, depending on whether you want to talk about the UI or the property of the flow.